server/server.go

```diff
@@ -295,26 +295,22 @@ func (s *Server) IsClosed() bool {
 	return atomic.LoadInt64(&s.isServing) == 0
 }
 
-var timeMonitorOnce sync.Once
-
 // Run runs the pd server.
 func (s *Server) Run(ctx context.Context) error {
-	timeMonitorOnce.Do(func() {
```

Comment:
why remove `do once`?

Reply:
After the context is canceled, the goroutine will exit. It seems no need to use it?